model/BUILD.bazel

```diff
@@ -1,270 +1,4 @@
-# TODO(#1532): Rename file to 'BUILD' post-Gradle.
 """
-This library contains all protos used in the app and is a dependency for all other modules.
-In Bazel, proto files are built using the proto_library() and java_lite_proto_library() rules.
-The proto_library() rule creates a proto file library to be used in multiple languages.
-The java_lite_proto_library() rule takes in a proto_library target and generates java code.
+Temporary package for defining all protos used in the codebase. Eventually, these will be moved to
+more targeted data directories post-Gradle.
```

Comment:
Do we need this file?

Reply:
We do since there are .bzl files in this directory & Bazel only supports importing .bzl files from an actual package directory.

Reply:
Sounds good.